.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        terraform_version: [0.12.29, latest]
+        terraform_version: [0.12.29, 0.13.6]
```

Comment:
The latest version(0.14.*) of terraform now has new formatting standards that aren't compatible with the older versions.... Limiting this to 0.13.6 so that the CI Tests pass.